pkgs/top-level/python-packages.nix

```diff
@@ -15814,7 +15814,7 @@ self: super: with self; {
 
   tree-sitter = callPackage ../development/python-modules/tree-sitter { };
 
-  tree-sitter0_21 = callPackage ../development/python-modules/tree-sitter0_21 { };
+  tree-sitter_0_21 = callPackage ../development/python-modules/tree-sitter/0_21.nix { };
```

Comment:
Would have been ideal to add an entry in python-aliases for the rename.

Reply:
Certainly, but it's an internal package and I didn't want to pollute the namespace too much.

Reply:
> > Would have been ideal to add an entry in python-aliases for the rename.
> Certainly, but it's an internal package and I didn't want to pollute the namespace too much.

Also, that attribute was there for a ~month or less, and it is an outdated version, so I barely think anyone had started using it externally.